src/parser.act

```diff
@@ -362,6 +435,26 @@
 		ast->exclusive_minimum = true;
 	@};
 
+	<set-min-properties>: (n :number) -> () = @{
+                ast_count c = (ast_count)@n;
```

Comment:
Check for negative values here, ala `<set-multiple-of>`. Ditto `<set-max-properties>` below.

Reply:
According to the AST, a max of 0 means unbounded, whereas in the json schema spec, a max of 0 means none. So that needs special handling, too. Does the AST have a way to express zero properties?

Reply:
The AST has no way to represent zero properties right now.  I could break the KWS_MINMAX_PROPERTIES (and other associated flags) into MIN and MAX versions.  Then unbounded would lack the KWS_MAX_PROPERTIES flag, instead of using zero.